playwright.config.ts

```diff
@@ -35,7 +35,7 @@ export default defineConfig({
   projects: [
     {
       name: 'chromium',
-      use: { ...devices['Desktop Chrome'] },
+      use: { ...devices['Desktop Chrome'], channel: 'chrome', },
```

Comment:
This was necessary to get the video test case running. 
Without it `video.play()` did not work with playwright.